corehq/motech/openmrs/tasks.py

```diff
@@ -252,9 +251,14 @@ def import_patients():
         import_patients_to_domain(domain_name)
 
 
-@task(queue='background_queue')
+@periodic_task(
+    run_every=crontab(**OPENMRS_ATOM_FEED_POLL_INTERVAL),
+    queue='background_queue'
+)
 def track_changes():
     """
     Uses the OpenMRS Atom Feed to track changes
     """
-    pass
+    domains = find_domains_with_toggle_enabled(toggles.OPENMRS_INTEGRATION)
+    for domain in domains:
+        poll_openmrs_atom_feeds(domain)
```

Comment:
Might be worth making this a task of its own so that a failure in one domain doesn't kill all.